doc/classes/VisualShaderNode.xml

```diff
@@ -27,6 +29,7 @@
 			<argument index="0" name="values" type="Array">
 			</argument>
 			<description>
+				Sets the default input ports using the [Array] filled with {index, value}. For example: {0, Vector3(0, 0, 0), 1, Vector3(0, 0, 0)}.
```

Comment:
Same as above, this should use `[code][index, value][/code]` to match the way the array looks like when printed (and in this case also when written in GDScript). Same for the example.